common/prettify.pm

```diff
@@ -1099,9 +1131,9 @@ sub new ($$$$$$)
             new Prettify::Totals_HTML ($basename), #Must be 2
             new Prettify::Config_HTML ($basename), #Must be 3
         );
-    
+
     if (!$skip_failed_test_logs) {
-        push @{$self->{OUTPUT}}, new Prettify::Failed_Tests_HTML ($basename, $buildname, $rev_link, $log_prefix); #Must be 4, if used
+        push @{$self->{OUTPUT}}, new Prettify::Failed_Tests_HTML ($basename, $buildname, $self->{FAILED_TESTS}, $rev_link, $log_prefix); #Must be 4, if used
```

Comment:
should this comment changed?

Reply:
Or removed

Reply:
There is logic that relies on this being #4 - similar to other reports above

Reply:
Maybe make the comment more clear then. I assumed it meant that it needed 4 variables passed to be used.

Reply:
I made it more clear, but there is also a context, which makes it even more understandable:

        @{$self->{OUTPUT}} =
            (
                new Prettify::Full_HTML ($basename),   #Must be at 0
                new Prettify::Brief_HTML ($basename),
                new Prettify::Totals_HTML ($basename), #Must be at 2
                new Prettify::Config_HTML ($basename), #Must be at 3
            );